src/include/meta.h

```diff
@@ -27,8 +27,9 @@
 #define WT_METAFILE_SLVG "WiredTiger.wt.orig" /* Metadata copy */
 #define WT_METAFILE_URI "file:WiredTiger.wt"  /* Metadata table URI */
 
-#define WT_LAS_FILE "WiredTigerLAS.wt"     /* Lookaside table */
-#define WT_LAS_URI "file:WiredTigerLAS.wt" /* Lookaside table URI*/
+#define WT_LAS_FILE "WiredTigerLAS.wt"       /* Lookaside table */
+#define WT_LAS_URI "file:WiredTigerLAS.wt"   /* Lookaside table URI*/
+#define WT_LAS_URI_HASHED 0xf2f88f2e3d0971dc /* Lookaside table URI Hashed */
```

Comment:
I used the LAS hash to determine if we had a dhandle that was pointing to the LAS file. There wasn't a way I could determine that reliably told you if you had a LAS dhandle. You could get the btree from the dhandle and check if the btree LAS flag was set however this isn't set in all code paths, so there are situations where you don't have that flag. I think we could spend time making that certain. Another option was to add a dhandle flag but that has a variety of issues.